WORKSPACE

```diff
@@ -0,0 +1,65 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```

Comment:
As per the @BenHenning comment about Bazel directory, we should have top-level documentation for WORKSPACE directory too.

> Please add a top-level documentation comment explaining what this directory represents at a high-level. It's conventional to document BUILD files to provide context since they're just as much code as other parts of the codebase.

Reply:
Ok will do!

Reply:
I pushed a comment to the WORKSPACE file - Please let me know if this is what you mean!

Reply:
This looks good. Just a nit change, I think it will look good this way
```
'''
This file lists and imports all external dependencies needed to build Oppia Android.
'''
```

Reply:
Also, we can leave a single line space after the comments so it can give a sense of file's documentation else it will look like comment about the below line of code.

Reply:
Also, we can leave a single line space after the comments so it can give a sense of file's documentation else it will look like the comment about the below line of code.

Reply:
Done!